### PR TITLE
Update format-specifiers-in-cpp.md

### PR DESCRIPTION
**Pull request by hharshada**
Added new specifier information.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@hharshada : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/format-specifiers-in-cpp.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ The following tables describe the format specifiers that you can use in Visual S
 |nr|Suppress "Raw View" item|
 |nvo|Show "Raw View" item for numeric values only|
 |!|raw format, ignoring any data type views customizations|\<customized representation>|4|
+|handle|displays information about win32 handle|0x000000000000009c| Dispalys available useful information about handle such as Thread Id etc. |
```
Comment (Mikejo5000) [inline docs/debugger/format-specifiers-in-cpp.md:101]:
```suggestion
|handle|displays information about win32 handle|0x000000000000009c| Displays useful information about handle such as thread ID, etc. |
```

**Comment by PRMerger15** [comment]
@hharshada : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Mikejo5000 — CHANGES_REQUESTED** [review]
@hharshada A couple of minor edits. On GitHub, you can commit them from the **Files changed** tab. (After you commit, you can pull the change to your local copy. In the git command line, that would be something like "git pull origin patch-3", but it depends on your setup.)

**Comment by j-martens** [comment]
@hharshada - do you want to fix or close this pull request? Thank you!

**Comment by hharshada** [comment]
Hey Mike,

I think I made a suggested fix for this already.

File: docs/debugger/format-specifiers-in-cpp.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ The following tables describe the format specifiers that you can use in Visual S
 |nr|Suppress "Raw View" item|
 |nvo|Show "Raw View" item for numeric values only|
 |!|raw format, ignoring any data type views customizations|\<customized representation>|4|
+|handle|displays information about win32 handle|0x000000000000009c| Displays useful information about handle such as thread ID, etc. |
```
Comment (Mikejo5000) [inline docs/debugger/format-specifiers-in-cpp.md:101]:
```suggestion
|handle|Displays information about win32 handle|0x000000000000009c| Displays useful information about handle such as thread ID, etc. |
```

**Comment by PRMerger15** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Mikejo5000 — APPROVED** [review]
Thanks!

**Comment by Mikejo5000** [comment]
#sign-off

**Pull request merged**